### ISSUE TITLE
retain timeline's expand/contract slider in X-sheet mode

### ISSUE THREAD

**Issue by IgnacioFer** [tahoma2d/tahoma2d#746]
It would be great to retain the expand/contract timeline slider when using the vertical X-sheet mode. The slider could be located horizontally at the bottom of the X-sheet but now affected the vertical expansion/compression of the X-sheet.

I know most programs use horizontal timelines, with the X-sheet being more of a legacy feature, but it's great to have the extra vertical drawing space when animating vertically-proportioned characters (like humans!) on a not-so-huge display (a 16'' tablet). For that reason too it'd be good if the 'New Layer' and 'Reframe on 1s', etc toolbar stayed on top of the X-sheet rather than remaining at the bottom of the screen.

**Comment by artisteacher** [comment]
Hopefully this can be ported over from OT for the next release: https://github.com/opentoonz/opentoonz/pull/3944

>  but is there a way to get rid of the Command Bar at the bottom of the screen

In the 2D room? Just unlock the workspace (top-right corner) and grab the "handle" on the left of the command bar to undock it. Afterwards you should be able to close the command bar. I do recommend always leaving the workspace locked when you are done renovating.

**Comment by IgnacioFer** [comment]
Yes, thank you, I managed to get it where I wanted after some struggling - and I'll leave it locked now. I'm trying to see if I can put only the Toolbar and the Viewer on the Cintiq and all the rest on the monitor - seems to be working.

**Comment by artisteacher** [comment]
When I do a dual monitor setup with an iPad, I leave the normal setup on the main monitor & drag a floating comboviewer to the iPad - that way I have easy access to the tools/tool options and viewer without any additional setup.

**Comment by IgnacioFer** [comment]
> When I do a dual monitor setup with an iPad, I leave the normal setup on the main monitor & drag a floating comboviewer to the iPad - that way I have easy access to the tools/tool options and viewer without any additional setup.

Comboviewer! That works much better, thank you, together with a narrow floating x-sheet to the side. Only drawback seems to be that when using comboviewer the playback slows down a lot? From being able to keep up with 24fps down to 19fps in my case. 
Also, I've noticed that if I shut T2D down without closing down the comboviewer it comes back up in the same place/size when restarting T2D, which is great. What is not so great is that if I turn off the Cintiq and open T2D, the playback is still slowed down as though the comboviewer was still running in the disconnected Cintiq - despite the computer having switched automatically from multiple to single monitor...? Something similar was happening when splitting T2D between two monitors- it does not seem to reset itself to single monitor if it was last shut down in a two monitor situation.

**Comment by IgnacioFer** [comment]
> Hopefully this can be ported over from OT for the next release: [opentoonz/opentoonz#3944](https://github.com/opentoonz/opentoonz/pull/3944)

I just realized that Blender uses the zoom controllers where you have a dot at either end of the slider bar - I have used this a lot in Premiere Pro and it's a great way to navigate timelines/x-sheets.

As Blender is open source and T2D is also open source freeware, would it be possible to take this from Blender? (Asked by someone who has 0 knowledge of coding)
